man/systemd.network.xml

```diff
@@ -992,6 +992,24 @@
             <para>Specifies the outgoing device to match. The outgoing interface is only available for packets originating from local sockets that are bound to a device.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SourcePort=</varname></term>
+          <listitem>
+            <para>Specifies the source port range match in fib rules. Default to unset.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>DestinationPort=</varname></term>
+          <listitem>
+            <para>Specifies the destination port range match in fib rules. Default to unset.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Protocol=</varname></term>
+          <listitem>
+            <para>Specifies the protocol range match in fib rules. A number one of TCP (6), UDP(17) or SCTP(132). Default to unset.</para>
```

Comment:
"protocol range"???

I think this should take a name of protocol, not a number. How about using `socket_protocol_from_name()`?